internal/controller/streams.go

```diff
@@ -37,16 +37,16 @@ const (
 )
 
 type streamClient struct {
-	ciliumNamespace           string
-	conn                      *grpc.ClientConn
-	client                    pb.KubernetesInfoServiceClient
-	disableNetworkFlowsCilium bool
-	falcoEventChan            chan string
-	flowCollector             pb.FlowCollector
-	logStream                 pb.KubernetesInfoService_SendLogsClient
-	networkFlowsStream        pb.KubernetesInfoService_SendKubernetesNetworkFlowsClient
-	resourceStream            pb.KubernetesInfoService_SendKubernetesResourcesClient
-	configStream              pb.KubernetesInfoService_GetConfigurationUpdatesClient
+	ciliumNamespace    string
+	conn               *grpc.ClientConn
+	client             pb.KubernetesInfoServiceClient
+	falcoEventChan     chan string
+	IPFIXCollectorPort string
```

Comment:
Nit _(not blocking the PR)_: Do you really need to export this field?
```suggestion
	ipfixCollectorPort string
```